zcash_client_sqlite/src/wallet/init.rs

```diff
@@ -805,8 +806,10 @@ mod tests {
         assert_eq!(rows.len(), expected_tables.len());
         for (actual, expected) in rows.iter().zip(expected_tables.iter()) {
             assert_eq!(
-                re.replace_all(actual, " "),
-                re.replace_all(expected, " ").trim(),
+                re.replace_all(&re_paren.replace_all(actual, " $1 "), " ")
+                    .trim(),
```

Comment:
This is now complex enough that we should have a `normalize` helper closure just underneath the regex definitions to deduplicate it, with documentation of what it is doing.

Reply:
The helper closure was introduced, but it is missing documentation (which I want because the `re_parens` regex is very difficult to parse visually, and it's not clear why this is needed and that it's not just stripping out all the parens and potentially introducing bugs in change detection).